home.html

```diff
@@ -0,0 +1,132 @@
+        <h3>Serums</h3>
+        <p class="explication">By using our serums, your skin will be more radiant,full of colour and it'll be less dry.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/bodylotion.png">
+        <h3>Lotions</h3>
+        <p class="explication">Usse our body lotions daily and you'll notice a difference in your skin's hydration levels.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/nails.png">
+        <h3>Nails</h3>
+        <p class="explication">Use our nail products to ensure that your nails are in good condition and the skin on hands and feet is well cared.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/hair.png"> 
+        <h3>Hair</h3>
+        <p class="explication">Our hair products will strengthen your hair follicles and enhance hair growth.</p>
+      </div>
+    </div>
+    <a class="button">View more products</a>
+  </section>
+  <div class="artists" id="Artists">
+    <h2 class="arts">Our Beautician Artists</h2>
+    <hr class="title-underline">
+    <div class="artist-wrapper"></div>
+    <!--
+      <div class="artist">
+          <img class="artist-image" src="images/esthetician3.png">
+          <div class="art-desc">
+            <h4>Marleine Tshel</h4>
+            <figcaption> <em>Internatioal esthetician and over 8 years of experience.</em></figcaption>
+            <hr class="artist-underline">
+            <p class="artist-paragraph">Finished her esthetician studies and got her certificate in 2012. She provides services for your skin care needs.</p>
+          </div>
+        </div>
+        <div class="artist">
+          <img class="artist-image" src="images/Makeup artist .png">
+          <div class="art-desc">
+            <h4>Anabelle Slovy</h4>
+            <figcaption><em>Part of the DMFA Makeup Academy Tutors at Harvard University</em></figcaption>
+            <hr class="artist-underline">
+            <p class="artist-paragraph">Anabelle have always been passionate about art and one of the most exquisite expressions of makeup. </p>
+          </div>
+        </div>
+        </div> --> 
+    </div>
+    <button class="artist-button">MORE <i class="fa fa-chevron-down"></i></button>
+  </div>
+  <div class="PARTNER">
+    <h2>Partners</h2>
+    <hr class="title-underline">
+    <div class="PARTNERS">
+      <img class="partner-img" src="images/partner1.png">
+      <img class="partner-img" src="images/partner2.png">
+      <img class="partner-img" src="images/partner3.png">
+      <img class="partner-img" src="images/partner4.png">
+      <img class="partner-img" src="images/partner5.png">
+    </div>
+  </div>
+  <div class="Footer">
+    <img class="LOGO2"src="images/our logo.png">
+    <p class="footer2">@Flawless <br> Some Rights Reserved.</p>
+  </div>
+
+  <script src="js/script.js"></script>
+</body>
+</html>
```

Comment:
please kindly implement these changes for your mobile version of the home page :

- [ ] Please increase the margin before the `view more products` button
- [ ] kindly implement the `more` button to display more makeup artist as it is in the design 😄